tests/test_app.py

```diff
@@ -0,0 +1,35 @@
+import os
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+from stanley.app import team_command, request_feedback_command
+
+
+@pytest.fixture
+def flask_app(monkeypatch):
```

Comment:
It feels a bit wrong to use monkeypatch to have a testing setup for a flask app. Did you check out their testing advices? https://flask.palletsprojects.com/en/1.1.x/testing/ There you can tamper with the config as well.

Reply:
In this case it needs to be like this because I'm mocking an environment variable `FLASK_APP`, used to specify how to load the application (more details [here](https://flask.palletsprojects.com/en/1.0.x/cli/#application-discovery)). As far as I understood, `FLASK_APP` isn't part of the app object configuration like `DATABASE`.

The goal here is mocking the value of `FLASK_APP` for a command like: `FLASK_APP=stanley/app.py flask team-command`.

Reply:
@amureki, would you know something about it?

Reply:
I am not sure I get it, won't test run successfully without `flast_app` fixture? Let's chat a bit more about it after lunch. 😉

Reply:
Are you sure?

```
tests/test_app.py:21: in test_team_command
    assert result.exit_code == 0
E   assert 2 == 0
E    +  where 2 = <Result SystemExit(2,)>.exit_code
_______________________________________________ test_send_slack_message ________________________________________________
tests/test_app.py:34: in test_send_slack_message
    assert result.exit_code == 0
E   assert 2 == 0
E    +  where 2 = <Result SystemExit(2,)>.exit_code
```

Reply:
My mistake, I did run it with old environment. But now we found much cleaner solution, I love it!